generated/nifake/tests/test_session.py

```diff
@@ -477,35 +477,35 @@ def test_get_attribute_error(self):
                 assert e.code == test_error_code
                 assert e.description == test_error_desc
 
-    # Error descriptions
```

Comment:
Put back "# Error descriptions"?
The whole thing is sort of organized with these headers.

Reply:
Fixed